AUTHORS.md

```diff
@@ -4,4 +4,5 @@ The Image Map module would not be possible without the generous contributions of
 
 We specifically want to thank our developers Kyle Chesney kyle.chesney@ufl.edu, James Pence jrpence@ufl.edu, and Philip Chase pbc@ufl.edu for their contributions to the project.  Taryn Stoffs tls@ufl.edu and Jerri Johnston j.johnston@ufl.edu provided valuable input into the design and testing of the Image Map module.
```

Comment:
This list is incomplete, please refer to the contributors page for filling it out.